ENS/ens-beginners-guide.md

```diff
@@ -0,0 +1,222 @@
+# ENS: Beginner's Guide
+
+User experience is a key challenge when developing a dapp. Ethereum has complexities that should be hidden away to enable broad adoption. One such complexity, is an Ethereum address, it is long, unwieldy, and hard to remember.
+
+The Ethereum Name Service (ENS) is an incredibly useful tool for dapp developers. ENS is like DNS, in that it maps a memorable shortcut to an address. Using ENS we can map the friendly name `ethereum.eth` to the rather unfriendly `0xfB6916095ca1df60bB79Ce92cE3Ea74c37c5d359`. Subsequently, the friendly name can be used in-place of the address, making it easier to remember, and reducing the chance of errors. 
```

Comment:
@darrenlangley Avoid the "weasel words" as they don't really add much, for example.

```suggestion
The Ethereum Name Service (ENS) is a useful tool for dapp developers. ENS is like DNS, in that it maps a memorable shortcut to an address. Using ENS we can map the friendly name `ethereum.eth` to the rather unfriendly `0xfB6916095ca1df60bB79Ce92cE3Ea74c37c5d359`. Subsequently, the friendly name can be used in-place of the address, making it easier to remember, and reducing the chance of errors. 
```

Reply:
And I'm fussy with this kind of stuff, but it does help writing and reading in the long run, so also try removing passive voice as much as possible, so for example, instead of 

> Subsequently, the friendly name can be used in-place of the address, making it easier to remember, and reducing the chance of errors. 

Try

> Subsequently, you (or "we" doesn't matter) can use the friendly name in-place of the address, making it easier to remember, and reducing the chance of errors.

Reply:
Have removed weasel word and reviewed text for passive voice.